Makefile

```diff
@@ -148,7 +148,7 @@ itest: itest-run
 
 itest-run:
 	@$(call print, "Running integration tests.")
-	$(GOTEST) ./itest -tags="$(ITEST_TAGS)" $(TEST_FLAGS) -logoutput -logdir=itest_logs
```

Comment:
It seems like this didn't generate any logs previously and the flags are not present anymore.